packages/core/template_project/app/build.gradle

```diff
@@ -72,9 +72,9 @@ android {
             resValue "string", "webManifestUrl", '<%= webManifestUrl %>'
         <% } %>
 
-        <% if (twaManifest.webShare) { %>
+        <% if (shareTarget) { %>
             // The data for the app to support web share target.
-            resValue "string", "webShare", twaManifest.webShare
+            resValue "string", "shareTarget", '<%= JSON.stringify(shareTarget) %>'
```

Comment:
Alternatively, we could add a `webShareTargetJson(): string` to `TwaManifest` and replace the call with that. Also maybe replace the `if` test with `hasShareTarget(): boolean`.

Reply:
About the JSON, done. 
About the if: It wouldn't make the code better I think.

Reply:
I'm fine with both of the options. I think they are both serves us the same

Reply:
@PEConn the reason I usually add to `twaManifest` because the template is more error prone - code is harder to read and we don't get any kind of type checking. It's also easier if we want to change the implementation in the future, but still have a method that returns a `string`. As I mentioned when I approved the request, this is more a suggestion and I'm happy either way.